pnpm-lock.yaml

```diff
@@ -3225,7 +3231,7 @@ packages:
     dependencies:
       '@astro-community/astro-embed-twitter': 0.1.3_astro@packages+astro
       '@astro-community/astro-embed-youtube': 0.1.1_astro@packages+astro
-      astro: link:packages/astro
+      astro: link:packages\astro
```

Comment:
not sure why these changed, can you fix them?

Reply:
Thanks, I will fix them.